src/components/shared/CallNodeContextMenu.js

```diff
@@ -493,6 +508,22 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
           content: 'Focus on subtree only',
         })}
 
+        {hasCategory
+          ? this.renderTransformMenuItem({
+              l10nId: 'CallNodeContextMenu--transform-focus-category',
+              l10nProps: {
+                vars: { categoryName },
+                elems: { strong: <strong /> },
+              },
+              shortcut: 'g',
+              icon: 'Focus',
+              onClick: this._handleClick,
+              transform: 'focus-category',
+              title: '',
```

Comment:
(not for this PR passing-by comment) why are we still passing the title?? :-)

Reply:
Copy pasta, I suppose...